aws-credentials/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+user: ubuntu
+group: ubuntu
+account_type: [default]
```

Comment:
Should this stay "default" by default? I actually don't know if I agree or disagree with it being default here. I'd lean towards it being maybe dangerous though since it has the capacity to break things if left as default.

Reply:
I initially suggested default (instead of restricted) because that was a specific use case, and I was trying to make this more generic, but I suppose we could make this default to `chia` or something equally generic, vs default